app/src/main/java/com/infomaniak/swisstransfer/ui/images/illus/onboarding/StorageBoxPileDark.kt

```diff
@@ -0,0 +1,632 @@
+package com.infomaniak.swisstransfer.ui.images.illus.onboarding
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.EvenOdd
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.group
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infomaniak.swisstransfer.ui.images.AppImages
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+
+val AppIllus.StorageBoxPileDark: ImageVector
+    get() {
+        if (_storageBoxPileDark != null) {
+            return _storageBoxPileDark!!
+        }
+        _storageBoxPileDark = Builder(
+            name = "StorageBoxPileDark",
+            defaultWidth = 289.0.dp,
+            defaultHeight = 258.0.dp,
+            viewportWidth = 289.0f,
+            viewportHeight = 258.0f
+        ).apply {
+            path(
+                fill = SolidColor(Color(0xFFA98116)),
+                stroke = null,
+                strokeLineWidth = 0.0f,
+                strokeLineCap = Butt,
+                strokeLineJoin = Miter,
+                strokeLineMiter = 4.0f,
+                pathFillType = EvenOdd
+            ) {
+                moveTo(97.98f, 94.05f)
+                curveToRelative(0.78f, 0.68f, 3.34f, 1.72f, 4.57f, 1.98f)
+                curveToRelative(-1.29f, -0.57f, -3.01f, -1.2f, -3.95f, -1.99f)
+                curveToRelative(-4.49f, -3.76f, -1.84f, -10.04f, 3.95f, -9.12f)
+                curveToRelative(1.06f, 0.17f, 2.86f, 1.02f, 4.31f, 1.73f)
+                curveToRelative(-0.97f, -0.69f, -3.81f, -2.05f, -4.84f, -2.28f)
+                curveToRelative(-4.83f, -1.07f, -9.7f, 4.74f, -4.04f, 9.68f)
+            }
+            group {
```

Comment:
This group looks like it can be removed.

Reply:
About groups in generaé, I have answered to the same comment here: https://github.com/Infomaniak/android-SwissTransfer/pull/222#discussion_r1870905083